raiden-ts/CHANGELOG.md

```diff
@@ -62,18 +74,22 @@
 [#2312]: https://github.com/raiden-network/light-client/pull/2312
 
 ## [0.12.0] - 2020-10-22
+
```

Comment:
Where comes this mass of new lines from? Please don't commit changes caused by custom linters we don't have at the repository.

Reply:
This is my prettier plugin in vscode i guess i need to disable it while making changes to CHANGELOG

Reply:
Yes, please. We might add Markdown formatting to the project in future.

Reply:
Done by @agatsoh